lib/request/sfWebRequest.class.php

```diff
@@ -148,7 +155,14 @@ public function initialize(sfEventDispatcher $dispatcher, $parameters = array(),
       $this->setMethod(self::GET);
     }
 
-    $this->postParameters = get_magic_quotes_gpc() ? sfToolkit::stripslashesDeep($postParameters) : $postParameters;
+    if (version_compare(PHP_VERSION, '5.4.0-dev', '<') && get_magic_quotes_gpc())
+    {
+      $this->postParameters = sfToolkit::stripslashesDeep($postParameters);
+    }
+    else
+    {
+      $this->postParameters = $postParameters;
+    }
```

Comment:
Why wouldn't you keep it in PSR codestyle?

```
        if (version_compare(PHP_VERSION, '5.4.0-dev', '<') && get_magic_quotes_gpc()) {
            $this->postParameters = sfToolkit::stripslashesDeep($postParameters);
        } else {
            $this->postParameters = $postParameters;
        }
```

Reply:
Oh sorry, that lead to a lot of unrelated changes, I see symfony1 has its own code style.
Would you mind reverting it?